### ISSUE TITLE
Is this abandoned?

### ISSUE THREAD

**Issue by dbalatero** [bling/fzf.el#60]
Hey @bling,

No worries either way, but wanted to see if this library is abandoned or not? I made a bunch of changes on [my fork](https://github.com/dbalatero/fzf.el), as the issues section here seems quiet and the last commit was in 2018.

**Comment by bling** [comment]
i wouldn't say it's abandoned, but it's also true i don't have much time these days to maintaining the project.  i just merged a bunch of low-risk PRs.

if you, or anyone else is interested in maintaining the project, please let me know.

**Comment by ssnnoo** [comment]
will you combine the two again or which one should I use?

**Comment by bling** [comment]
this one is more up to date at this point and after #63 was merged it also provides significantly more functionality.

**Comment by ssnnoo** [comment]
thanks!

**Comment by sayeefm0** [comment]
Trying here before opening a new issue - I recently forked this to make some changes / clean up the code before I noticed this issue. I'd be happy to take over ownership of the package. Let me know if you're still interested in handing it off.

**Comment by bling** [comment]
@sayeefm0 can u please open a PR and go from there?  thanks.

**Comment by sayeefm0** [comment]
Sure! I'm working on my changes atm. I'll send it over when I'm done. It's possible that the direction I'm taking is different than what you wanted, in which case I can also split off and make an entirely separate package.

**Comment by sayeefm0** [comment]
I figure you're busy, but in case you just didn't see, I opened a PR with my current changes. I'll keep pushing my changes up so the PR will be up to date.

**Comment by neojski** [comment]
@pierre-rouleau seems to be maintaining the project now.

**Comment by pierre-rouleau** [comment]
I would say I'm maintaining it.  I started using it and want to get the most of it before I'll no longer have time to work on it;-)
